<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player</class>
 <widget class="QDialog" name="Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>25</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flight Player</string>
  </property>
  <layout class="QVBoxLayout" name="toolBarLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QSlider" name="slider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbTime">
       <property name="text">
        <string>00:00:00/00:00:00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="aPlay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/standard-icons.qrc">
     <normaloff>:/icons/old/transport_play.png</normaloff>:/icons/old/transport_play.png</iconset>
   </property>
   <property name="text">
    <string>Play/Pause</string>
   </property>
   <property name="toolTip">
    <string>Play/Pause telemetry data</string>
   </property>
  </action>
  <action name="aStop">
   <property name="icon">
    <iconset resource="../../resources/standard-icons.qrc">
     <normaloff>:/icons/old/transport_stop.png</normaloff>:/icons/old/transport_stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources/standard-icons.qrc"/>
  <include location="../../resources/standard-icons.qrc"/>
  <include location="../../resources/standard-icons.qrc"/>
  <include location="../../resources/standard-icons.qrc"/>
  <include location="../../resources/standard-icons.qrc"/>
 </resources>
 <connections/>
</ui>
